packages/@aws-cdk/aws-iam/README.md

```diff
@@ -249,6 +249,18 @@ const newPolicyDocument = PolicyDocument.fromJson(policyDocument);
 
 ```
 
+### Policy Statement
+
+As per the doc of [IAM policy grammar](https://docs.aws.amazon.com/en_us/IAM/latest/UserGuide/reference_policies_grammar.html#policies-grammar-bnf), the action and resource blocks are mandatory. Hence, the new policy stament using `PolicyStatementProps` can be created as:
```

Comment:
Not sure this README section is helping a lot. What kind of information do you think people are missing?

Or did you write this section just to appease the linter?

Reply:
Yes, I just added it to appease the linter. Is there any better way to handle this?

Reply:
Ha, yes, we can add a tag to silence it.